website/docs/components/form/text-input/partials/code/component-api.md

```diff
@@ -1,43 +1,46 @@
-The `Form::TextInput` component has two different variants, with their own APIs:
+## Component API 
 
-*   `Form::TextInput::Base` - the "basic" component: just the `<input>` control
-*   `Form::TextInput::Field` - the "field" parent component: the `<input>` control, with label, helper text and error messaging (in a wrapping container)
+The `Form::TextInput` component has two different variants, with their own APIs:
 
-#### Form::TextInput::Base
+- `Form::TextInput::Base` - the base component: the `<input>` control
+- `Form::TextInput::Field` - the field parent component: the `<input>` control, with label, helper text, and error messaging (in a wrapping container)
 
-Here is the API for the "base" component:
+### Form::TextInput::Base
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="type" @type="enum">
-    Sets the native HTML `type` of the `<input>`. This list covers all the official types (see [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input))
+    Sets the native HTML `type` of the `<input>`. This list covers all the official types (see [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)).
   </C.Property>
   <C.Property @name="value" @type="string|number|date">
     The input control's value.
   </C.Property>
   <C.Property @name="isInvalid" @type="boolean">
-    It applies an "invalid" appearance to the control (_notice: this does \_not\_ modify its logical validity_).
+    It applies an "invalid" appearance to the control, but doesn't modify its logical validity.
   </C.Property>
   <C.Property @name="width" @type="string" @valueNote="any valid CSS width (px, rem, etc)">
-    _Notice: by default the `<input>` has a `width` of `100%` applied to it, so it fills the parent container. If a `@width` parameter is provided then the control will have a fixed width._
+    By default the `<input>` has a `width` of `100%` applied to it, so it fills the parent container. If a `@width` parameter is provided then the control will have a fixed width.
   </C.Property>
   <C.Property @name="...attributes">
-    `...attributes` spreading is supported on this component. _Notice: the attributes will be applied to the `<input>` element. This means you can use all the standard HTML attributes of the `<input>` element and all the usual Ember techniques for event handling, validation, etc._ _Some examples of HTML attributes that you will likely use: `id`, `name`, `value`, `placeholder`, `disabled`, `readonly`, `required` ([see whole list here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes)) and some examples of Ember modifiers: `\{{on "input" [do something]}}`, `\{{on "change" [do something]}}`, `\{{on "blur" [do something]}}`._
+    `...attributes` spreading is supported on this component. 
```

Comment:
```suggestion
This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
```

Reply:
Will try to use this phrasing consistently across components